WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -3,8 +3,8 @@
 require 'csv'
 
 class AdminController < ApplicationController
-  before_action :authenticate_user!
-  before_action -> { redirect_to_root_unless_user(:can_admin_results?) }, except: [:all_voters, :leader_senior_voters]
+  before_action :authenticate_user!, except: [:person_data]
```

Comment:
Given we no longer authorize user in case of `person_data`, I think it would make sense to move this endpoint to some API controller. Thoughts?

Reply:
Ah yeah, totally. Turns out we already have a good api for this over in `/api/v0/users/:wca_id`.

I modeled this code after our existing admin `edit_person` page: https://staging.worldcubeassociation.org/admin/edit_person. That page uses the `/admin/person_data` route because it needs the person's private DOB as well. One possible future cleanup would be to update the `edit_person` page to use our `/api/v0/users/:wca_id` route instead, which would require some way for that api to sometimes return private information like DOBs when the requester is a website admin. (The check would actually be a little more involved than that, and I think we'd need to create an oauth scope for this, because we wouldn't want some WRT member to oauth into some third party website and for that third party website to suddenly have access to dobs).